pkg/features/kruise_features.go

```diff
@@ -152,16 +152,16 @@ var defaultFeatureGates = map[featuregate.Feature]featuregate.FeatureSpec{
 	KruisePodReadinessGate:                    {Default: false, PreRelease: featuregate.Alpha},
 	PreDownloadImageForInPlaceUpdate:          {Default: false, PreRelease: featuregate.Alpha},
 	CloneSetPartitionRollback:                 {Default: false, PreRelease: featuregate.Alpha},
-	ResourcesDeletionProtection:               {Default: true, PreRelease: featuregate.Alpha},
-	WorkloadSpread:                            {Default: true, PreRelease: featuregate.Alpha},
-	PodUnavailableBudgetDeleteGate:            {Default: true, PreRelease: featuregate.Alpha},
+	ResourcesDeletionProtection:               {Default: true, PreRelease: featuregate.Beta},
```

Comment:
ResourcesDeletionProtection is a beta feature, but it need hook the deletion of namespace/crd and built-in workloads, plz close the featuregate by default